s3tests_boto3/functional/test_s3.py

```diff
@@ -9412,20 +9413,18 @@ def setup_lifecycle_expiration(client, bucket_name, rule_id, delta_days,
 
 def check_lifecycle_expiration_header(response, start_time, rule_id,
                                       delta_days):
-    print(response)
-    print(response['ResponseMetadata']['HTTPHeaders'])
+    expr_exists = ('x-amz-expiration' in response['ResponseMetadata']['HTTPHeaders'])
+    if (not expr_exists):
+        return False
+    expr_hdr = response['ResponseMetadata']['HTTPHeaders']['x-amz-expiration']
```

Comment:
I'm trying to remember, can't these throw an (unhandled) exception?

Reply:
from the code it looks like the tests are expecting to get bool and to compare it to True or False

For example:

eq(check_lifecycle_expiration_header(response, now, 'rule1', 1), True)

Reply:
oh, ok, you checked for x-amz-expiration on line 9416